packages/cdk/big-decimal/src/js-big-decimal/compare-to.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */

```

Comment:
try refactor without eslint disable

Reply:
I will, almost done that
 for now I moved all tests and glad it works
 continue refactoring